nemo_skills/pipeline/utils.py

```diff
@@ -498,6 +498,14 @@ def get_env_variables(cluster_config):
         elif env_var in default_factories:
             env_vars[env_var] = default_factories[env_var]()
             logging.info(f"Adding optional environment variable {env_var} from environment")
+        elif "=" in env_var:
+            if env_var.count("=") == 1:
+                env_var, value = env_var.split("=")
+            else:
+                env_var, *value = env_var.split("=")
```

Comment:
What's the use-case for multiple =? Should we instead error-out for that?

Reply:
I think i had a use case when passing dict like data via env but its so obscure that it doesn't make sense to support it.